apps/controllo-ore-x/src/app/_core/interfaces/i-menu-sections.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface IMenuSections {
+  label: string;
+  routerLink?: string;
```

Comment:
```suggestion
  routerLink: string;
```
This is found in all the current menus